db/seeds.rb

```diff
@@ -27,6 +27,19 @@
 Topic.find_or_create_by(name: "noise")
 Topic.find_or_create_by(name: "health")
 
+Report.all.each do |report|
```

Comment:
This has little impact on this example, but in general it's good practice to replace`all.each` with `find_each`